source/process/handbook.rst

```diff
@@ -34,3 +34,18 @@ Mattermost Team Edition under MIT license and the Mattermost open source project
 Mattermost Enterprise Editions, which are commercial extensions built on top of the Mattermost open source projects, are developed, supported and sold by Mattermost, Inc.
 
 For more information, see our `Product Overview <https://docs.mattermost.com/overview/product.html>`__.
+
+Company 
+-------------- 
+
+Mattermost is an open source, remote-first, communities-centered company based in Palo Alto, California and headquartered on the internet. 
+
+**Open source** means that by default we make our technology, business process and source code available to the public. We develop a small portion as proprietary technology, built upon our open source work, to license for subscription fees that enable more high quality open source work to be produced. Since the start of the Mattermost open source project in 2015, we have used this model to develop effective open source solutions for the world to use. This model is sometimes called `Thin Open Core <https://medium.com/open-consensus/2-open-core-definition-examples-tradeoffs-e4d0c044da7c>`_.
+
+**Remote-first** means that by default our staff works anywhere in the world where a) we have support for quality video calling and b) where we can work in a legally appropriate environment. Mattermost was born in an extraordinary age where organizations can attract, hire and enable remote teams--working from homes, coffee shops and co-working spaces across timezones--to produce better technology and business results in less time than purely office-based teams, while maintaining security and compliance standards. 
+
+Remote-first culture flourishes when we share one simple principle: **Courtesy**. Courtesy means Mattermosters are thoughtful about keeping our communities appropriately informed, following etiquette for discussions, calls and video meetings, and giving and receiving feedback on how to work better together. Courtesy also means we gladly accommodate those who prefer to work out of a dedicated office. We are remote-first, not remote-only.
+
+**Communities-centered** means we define our success in the context of the success of our `communities <https://docs.mattermost.com/process/community-overview.html>`_: users, customers, implementers, resellers, technology partners, contributors, and colleagues. The success of each community is owned by a member of the Mattermost leadership team. The plural definition of "communities" is intended to avoid unconsciously marginalizing downstream stakeholders. Referring to open source contributers as "the community", as if no other communities should be recognized, is like saying "a developer should format his code according to the style guide of the project". These statements may sound normal to the speaker, but they can unintentionally marginalize others.
```

Comment:
> Referring to open source contributers as "the community", as if no other communities should be recognized, is like saying "a developer should format his code according to the style guide of the project". These statements may sound normal to the speaker, but they can unintentionally marginalize others.

1. Contributors is spelt wrong here 
2. I feel like this sentence is a little strange (like almost but not quite passive aggressive?)
3. I think the example used here  "a developer should format his code according to the style guide of the project" isn't clear enough to get your point across. I had to read it twice to understand the point, and I think there are many other people who will need to read it more times than that.

Reply:
Thanks @lfbrock, deleted the line. It's intention was to make people think, but it fails the "fast, obvious, forgiving" test, so removed it.